serv/server.py

```diff
@@ -3,15 +3,27 @@
 from flask import Flask, request, jsonify
 from flask_cors import CORS
 import serv.semrider as sm
+import os
 
+embed_file = "serv/res/embed_prod_v02_rc.pkl"
+meta_file = "serv/res/meta_prod_v02_rc.pkl"
+
+# Clean up 
+clean_up = True
```

Comment:
Move io operations to iops, don't do it here. Only core server/routing logic should exist here.